src/azure-cli/azure/cli/command_modules/acs/_helpers.py

```diff
@@ -9,8 +9,13 @@
 
 
 def _populate_api_server_access_profile(api_server_authorized_ip_ranges):
+    if api_server_authorized_ip_ranges == "":
+        authorized_ip_ranges = []
+    else:
+        authorized_ip_ranges = [ip.strip() for ip in api_server_authorized_ip_ranges.split(",")]
+
     return ManagedClusterAPIServerAccessProfile(
```

Comment:
This would overwrite other properties in ManagedClusterAPIServerAccessProfile for eg. enablePrivateCluster? Won't this be a problem?

Reply:
Yup, you are right - will update

Reply:
Updated